packages/jest-snapshot/src/InlineSnapshots.ts

```diff
@@ -49,10 +49,15 @@ export function saveInlineSnapshots(
   snapshots: Array<InlineSnapshot>,
   prettierPath: Config.Path,
 ): void {
-  const prettier = prettierPath
-    ? // @ts-expect-error requireOutside Babel transform
-      (requireOutside(prettierPath) as Prettier)
-    : undefined;
+  let prettier;
```

Comment:
```suggestion
  let prettier: Prettier | null = null;
```

or something?

Reply:
Confirmed this works too, please accept if you'd prefer this.